internal/services/instances.go

```diff
@@ -534,3 +536,25 @@ func convertUnstructuredListIntoHelmCharts(unstructuredList *unstructured.Unstru
 
 	return helmChartList, nil
 }
+
+func concatenateCatalogServiceValues(catalogService models.CatalogService, name string) string {
+
+	type FixedValues struct {
+		ServiceName string `json:"serviceName,omitempty"`
+	}
+	var fixedValues FixedValues
+	var builder strings.Builder
+
+	fixedValues.ServiceName = name
+	// Build the concatenated string
+
+	builder.WriteString("\nserviceName: ")
+	builder.WriteString(fixedValues.ServiceName)
+	builder.WriteString("\n")
```

Comment:
To clarify, you are building a YAML fragment here, right ?

Reply:
Yes, that's it!

Reply:
In that case you may wish to know about the Go standard library yaml package which can do this for you from properly annotated structures. See https://pkg.go.dev/gopkg.in/yaml.v2#Marshal

Reply:
The main point of this is that the `yaml` package does all the work for you to make the result a proper YAML string, with all the necessary quoting and what not.

... Example code https://dev.to/sagartrimukhe/generate-yaml-files-in-golang-29h1

Reply:
Hi! It was implemented, thank you for your tips and examples, If you prefer I can move the struct `FixedValues` to the models package.